internal/pkg/mapper/sharedcode/helper/helper.go

```diff
@@ -77,75 +95,24 @@ func (h *SharedCodeHelper) GetTargetComponentId(sharedCodeConfig *model.Config)
 	return model.ComponentId(componentId), nil
 }
 
-func (h *SharedCodeHelper) GetSharedCodePath(object model.Object) (*model.Config, string, error) {
```

Comment:
To si vyziadalo prepisanie takmer celeho balicka `sharedcode/links`.
- funkcionalne testy sa nemenili vobec
- a povodne unit testy som zachoval (upravil) a doplnil som dalsie.